pandas/tests/generic/test_generic.py

```diff
@@ -88,6 +88,7 @@ def test_get_numeric_data(self, frame_or_series):
         tm.assert_equal(result, o)
 
         # non-inclusion
+        # https://github.com/pandas-dev/pandas/issues/50862
```

Comment:
sorry i wasn't clear, could you put this just above the line that does 
```
result.reset_index(drop=True)
```
(as that's the one which needs explaining)

Reply:
No problem, it's done now though, so I'm assuming that I don't need to do anything else, let me know if there is something else that I need to do though.